nft_ingester/src/json_worker.rs

```diff
@@ -341,6 +346,10 @@ impl JsonDownloader for JsonWorker {
             },
         }
     }
+
+    fn skips_refreshes(&self) -> bool {
```

Comment:
I do not think it requires a dedicated getter. Why can't we call `self.should_skip_refreshes` ?

Reply:
Nvm, in case it is requested because a caller is a generic then it's acceptable